scripts/delete_bundle.py

```diff
@@ -0,0 +1,51 @@
+import sys
+import argparse
+
+import logging
+import requests
+from azul import config
+
+logger = logging.getLogger(__name__)
+
+
+def main(argv):
+    parser = argparse.ArgumentParser(description='Delete bundles from Azul index.')
+    parser.add_argument('bundles',
+                        metavar='bundle_uuid.bundle_version',
```

Comment:
```suggestion
                        metavar='UUID.VERSION',
```

Convention is for meta vars to be all-caps.